po/meson.build

```diff
@@ -0,0 +1,37 @@
+dep_intl = []
+if get_option('nls')
+  inc_args = []
+  dep_intl = dependency('intl', required: false)
+  if not dep_intl.found()
```

Comment:
I might be able to get rid of all these conditions, and get it working on Windows by using the [proxy-libintl](https://github.com/frida/proxy-libintl) wrap maintained by @frida